app/src/controllers/uniqueErrors/sagas.js

```diff
@@ -55,18 +68,44 @@ function* fetchClusters(payload = {}) {
   } else {
     yield call(fetchParentLaunch, { payload: { project, launchId } });
   }
-  parentLaunch = yield select(launchSelector);
+
   const namespace = yield select(namespaceSelector);
   const query = yield select(queryParametersSelector, namespace);
-  yield put(
-    fetchDataAction(NAMESPACE)(
-      URLS.clusterByLaunchId(project, launchId, {
-        [PAGE_KEY]: query[PAGE_KEY],
-        [SIZE_KEY]: query[SIZE_KEY],
-        [SORTING_KEY]: query[SORTING_KEY],
-      }),
-    ),
+  const supportedIntegration = integrations.find(
+    (integration) =>
+      integration &&
+      integration.integrationType &&
+      integration.integrationType.details &&
+      integration.integrationType.details.metadata &&
+      integration.integrationType.details.metadata.supportedFeatures &&
+      integration.integrationType.details.metadata.supportedFeatures.includes('newErrors'),
   );
+
+  let url;
+  const requestParams = {};
+  if (supportedIntegration) {
+    url = URLS.projectIntegrationByIdCommand(
+      project,
+      supportedIntegration.id,
+      COMMAND_GET_CLUSTERS,
+    );
+    requestParams.method = 'PUT';
+    const uniqueErrorsParams = yield select(pagePropertiesSelector, NAMESPACE);
+    requestParams.data = {
+      launchId,
+      mode: uniqueErrorsParams.mode || ALL,
```

Comment:
May be in core we don't need to know about this parameter and its possible values.
Can we just spread this params? Will the BE accept request params with empty `mode`?